This patch was pulled from rt2500 cvs by Ulrich Plate <plate@gentoo.org>
to fix http://bugs.gentoo.org/show_bug.cgi?id=85079

It fixes kernel changes in kernels >2.6.11

--- rt2500-1.1.0/Module/rtmp_main.c.orig	2005-05-18 09:22:51.000000000 +0200
+++ rt2500-1.1.0/Module/rtmp_main.c	2005-05-18 09:23:12.000000000 +0200
@@ -100,7 +100,7 @@
 
     printk("%s %s %s http://rt2x00.serialmonkey.com\n", KERN_INFO DRV_NAME, DRV_VERSION, DRV_RELDATE);
 
-    print_name = pPci_Dev ? pPci_Dev->slot_name : "rt2500";
+    print_name = pPci_Dev ? pci_name(pPci_Dev) : "rt2500";
 
     // alloc_etherdev() will set net_dev->name
     net_dev = alloc_etherdev(sizeof(RTMP_ADAPTER));
